test/e2e/gardener/shoot/create_and_delete_hibernated.go

```diff
@@ -54,7 +54,8 @@ var _ = Describe("Shoot Tests", Label("Shoot", "default"), func() {
 })
 
 func verifyNoPodsRunning(ctx context.Context, f *framework.ShootCreationFramework) {
-	podsAreExisting, err := kubernetesutils.ResourcesExist(ctx, f.ShootFramework.SeedClient.Client(), corev1.SchemeGroupVersion.WithKind("PodList"), client.InNamespace(f.Shoot.Status.TechnicalID))
-	Expect(err).To(Succeed())
-	Expect(podsAreExisting).To(BeFalse())
+	podList := &metav1.PartialObjectMetadataList{}
+	podList.SetGroupVersionKind(corev1.SchemeGroupVersion.WithKind("PodList"))
+	ExpectWithOffset(1, f.ShootFramework.SeedClient.Client().List(ctx, podList, client.InNamespace(f.Shoot.Status.TechnicalID))).To(Succeed())
+	ExpectWithOffset(1, podList.Items).To(HaveLen(0))
```

Comment:
```suggestion
	ExpectWithOffset(1, podList.Items).To(BeEmpty())
```
Should have a similar output?